src/bosh-director/lib/bosh/director/api/config_manager.rb

```diff
@@ -13,6 +13,12 @@ def create(type, name, config_yaml, team_id = nil)
             config.save
           end
 
+          def deploy_config_enabled?
+            deploy_config = find(type: 'deploy', limit: 999)
```

Comment:
Is there any significance for choosing a limit of 999 here? It seems like you could do the (default) limit of 1, since you only care about the existence of the data, not the data itself.

Reply:
Thanks for the review! You are right. I have changed it.